examples/user_interfaces/navigation.py

```diff
@@ -0,0 +1,70 @@
+import matplotlib
+matplotlib.use('GTK3Cairo')
+# matplotlib.use('TkAGG')
+matplotlib.rcParams['toolbar'] = 'navigation'
```

Comment:
Since these are examples, I would suggest getting rid of extraneous commented out code, and also highlight important lines of code such as this one. For example, is it important that it gets called before importing pyplot? What is it for? Perhaps a short docstring at the top of this example would help explain its purpose/goal that it is demonstrating?